components/datetime/src/provider/date_time.rs

```diff
@@ -412,6 +406,9 @@ impl<'data> DateSymbols for provider::calendar::DateSymbolsV1<'data> {
             fields::FieldLength::Narrow => &self.eras.narrow,
             _ => &self.eras.abbr,
         };
+        if symbols.is_empty() {
+            return ""; // If symbols is empty, this calendar doesn't use eras (ex. Chinese)
+        }
```

Comment:
Issue: Not sure this is the right way to solve this problem. The era symbol _should never have been requested_ if the calendar doesn't have eras. The pre-existing fallback behavior of using the era code name (which is the calendar name) seems acceptable.